test/validation/AllObjects.spec.ts

```diff
@@ -165,16 +165,20 @@ function test<A extends { [_: string]: any }>(
                 arbitrary,
                 value => {
                     const key = pickFrom(keys);
-                    value[key] = null;
-                    return !isValid(validation.validate(value));
+                    // upcast to any so we can produce an invalid value.
```

Comment:
TypeScript 3.5 added stricter checks on assignments which broke this. Given that we are deliberately doing invalid stuff here, casting to `any` seemed like a reasonable option but I’m happy to learn others.